alpha/lib/model/StorageProfile.php

```diff
@@ -382,12 +382,6 @@ public function isValidFileSync(FileSyncKey $key)
 			return false;
 		}
```

Comment:
if $local is not in use anymore than remove it - list($kalturaFileSync,) = kFileSyncUtils::getReadyFileSyncForKey($key, true, false);